static/app/views/dashboardsV2/manage/dashboardList.tsx

```diff
@@ -69,7 +71,9 @@ function DashboardList({
       const dashboardDetail = await fetchDashboard(api, organization.slug, dashboard.id);
       const newDashboard = cloneDashboard(dashboardDetail);
       newDashboard.widgets.map(widget => (widget.id = undefined));
-      await createDashboard(api, organization.slug, newDashboard, true);
+
+      // -1 for project IDs means it should include all projects
+      await createDashboard(api, organization.slug, newDashboard, true, [-1]);
```

Comment:
should we reuse the const https://github.com/getsentry/sentry/blob/b519955596ed9d31f756f6a043721550f51d8263/static/app/constants/pageFilters.tsx#L26 here?

Reply:
Great idea, I'm using it now, thanks!